instructions/FOSS_staging_list.pl

```diff
@@ -128,14 +128,6 @@
          SysExec("cp -f build/zimlet/*.zip $CFG{BUILD_DIR}/zm-certificate-manager-admin-zimlet/build/zimlet");
       },
    },
-   {
-      "dir"         => "zm-clientuploader-admin-zimlet",
```

Comment:
we have removed this from here but we haven't written circleci workflows to generate these extension/zimlet so how are we planning to generate artefacts for release?

Reply:
Sure, we will add ci circle YML file for packages.

Reply:
@umagmrit Please take care it.